ovs/lib/albanode.py

```diff
@@ -320,6 +320,12 @@ def reset_osd(node_guid, osd_id, expected_safety):
             node.client.fill_slot(osd.slot_id, fill_slot_extra)
         except (requests.ConnectionError, requests.Timeout):
             AlbaNodeController._logger.warning('Could not connect to node {0} to (re)configure ASD'.format(node.guid))
+        except NotFoundError:
+            # Can occur when the slot id could not be matched with an existing slot on the alba-asd manager
+            # This error can be anticipated when the status of the osd would be 'missing' in the nodes stack but that would be too much overhead
+            message = 'Could not add a new osd. The requested slot: {0} could not be found'.format(osd.slot_id)
```

Comment:
Sentences with a colon in the middle of em, is kinda strange
`osd` is an abbreviation, so should be `OSD`